build/conda/installer/post_install.sh

```diff
@@ -1,39 +1,89 @@
 #!/bin/bash
 
-# Full path to the Python executable inside the constructed environment
+# Setup environment.
+export PATH="$PREFIX/bin:$PATH"
```

Comment:
same as the comment in the bat script. we need to make sure that all PATH stuff is setup correctly for the script to function properly, but we want to avoid altering the path on a system level.

Reply:
These PATH variables are only within the current session and script. Modifying PATH permanently requires writing to the actual shell profile.